.github/workflows/test.yml

```
@@ -1,22 +1,22 @@
-      - run: npm run all
-  test: # make sure the action works on a clean machine without building
+    - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683  # v4.2.2
+    - run: npm install
+    - run: npm run all
+  test:  # make sure the action works on a clean machine without building
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
-      - uses: ./
+    - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683  # v4.2.2
+    - uses: ./
```

actionlint (1.7.4)

missing input "image" which is required by action "Dive Action v2" defined at "./". all required inputs are "github-token", "image"

(action)